src/main/java/uk/gov/laa/ccms/caab/controller/AssessmentController.java

```diff
@@ -76,4 +94,113 @@ public String assessmentRemovePost(
   }
 
 
+  /**
+   * Displays the assessment page.
+   *
+   * @param assessment the assessment to remove.
+   * @param invokedFrom the page from which the assessment is being invoked.
+   * @param user the user making the request.
+   * @param session the session in which the request is being made.
+   * @param request the http servlet request being made.
+   * @return the view that displays the OPA assessment.
+   */
+  @GetMapping("/assessments")
+  public String assessmentGet(
```

Comment:
Please can these be compared to the files:

com.legalservices.laa.ccms.controller.iframeHoldingController In PUI

@RequestMapping(value = {"/renderOWD","/renderOWD.do"}, method = RequestMethod.GET)
  @Transactional
  public String editIFrameHolding(Map<String, Object> model,
      HttpServletRequest request,
      @RequestParam("invokedFrom") String invokedFrom,
      @RequestParam("assessmentType") String assessmentType,
      @RequestParam("ezgov_private_hiddenData_uniqueId") String uniqueID) {
      
Logic has been replicated up to the point of creating the means and merits data in the opa session tables. 
Anything todo with amendments or case data has been left as a todo as this implementation is only for initial applications.

The controller will redisplay the summary screen, still to implement and integrate fully with OPA to display the assessments.